test/client/spec/victory-core/victory-util/data.spec.js

```diff
@@ -141,8 +137,8 @@ describe("victory-util/data", () => {
           const data = createData([{ x: "kittens", y: 3 }, { x: "cats", y: 5 }]);
           const props = { data, x: "x", y: "y" };
           const expectedReturnWithEventKeys = [
-             { _x: 1, x: "kittens", xName: "kittens", _y: 3, y: 3 },
-             { _x: 2, x: "cats", xName: "cats", _y: 5, y: 5 }
+            { _x: 1, x: "kittens", xName: "kittens", _y: 3, y: 3, eventKey: 0 },
```

Comment:
These `eventKey` properties just got removed, which is why your specs are failing here